### PR TITLE
EDU-2741: add new top level cloud observability suggestions

### PR DESCRIPTION
**Pull request by axfelix**

### CODE REVIEW

**Comment by axfelix** [comment]
@fairlydurable This was a requested change from product to clarify the distinction between Cloud and SDK metrics which I've further edited; lmk if other changes are needed before merging.

File: docs/production-deployment/cloud/metrics/index.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 id: index
-title: Temporal Cloud observability through metrics
+title: Temporal Cloud Observability and Metrics
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/metrics/index.mdx:3]: Metrics is not on my capitalize list ¯\_(ツ)_/¯

File: docs/production-deployment/cloud/metrics/index.mdx [hunk 2]
```diff
@@ -27,8 +27,18 @@ tags:
   - temporal-cloud
 ---
 
-Beyond the [metrics](/references/sdk-metrics) provided by the Temporal SDKs, some key metrics exist only in Temporal Cloud.
-You can use your own observability tool to query an endpoint and review Namespace metrics.
+Temporal offers two distinct sources of metrics: [Cloud/Server Metrics](/production-deployment/cloud/metrics/reference) and [SDK Metrics](/references/sdk-metrics).
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/metrics/index.mdx:30]: Sentence works without "distinct", too.

File: docs/production-deployment/cloud/metrics/index.mdx [hunk 3]
```diff
@@ -27,8 +27,18 @@ tags:
...
+Each source provides options for levels of granularity and filtering, monitoring-tool integrations, and configuration.
+Before implementing observability for Temporal Cloud, decide what you need to measure for your use case. There are two primary use cases for metrics:
+- To measure the health and performance of Temporal-backed applications and key business processes.
+- To measure the health and performance of Temporal infrastructure and user provided infrastructure in the form of Temporal Workers and Temporal Clients.
+
+When measuring the performance of Temporal-backed applications and key business processes, you should rely on Temporal SDK metrics as a source of truth.
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/metrics/index.mdx:36]: This says "you should rely on" and then the next sentence almost contradicts that. Missing word? Intentional?

File: docs/production-deployment/cloud/metrics/index.mdx [hunk 4]
```diff
@@ -27,8 +27,18 @@ tags:
   - temporal-cloud
 ---
 
-Beyond the [metrics](/references/sdk-metrics) provided by the Temporal SDKs, some key metrics exist only in Temporal Cloud.
-You can use your own observability tool to query an endpoint and review Namespace metrics.
+Temporal offers two distinct sources of metrics: [Cloud/Server Metrics](/production-deployment/cloud/metrics/reference) and [SDK Metrics](/references/sdk-metrics).
+Each source provides options for levels of granularity and filtering, monitoring-tool integrations, and configuration.
+Before implementing observability for Temporal Cloud, decide what you need to measure for your use case. There are two primary use cases for metrics:
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/metrics/index.mdx:32]: Consider "Temporal Cloud observability"

Second sentence, if you want to get rid of passive voice, "Use metrics to" (and adjust bullets).

File: docs/production-deployment/cloud/metrics/index.mdx [hunk 5]
```diff
@@ -27,8 +27,18 @@ tags:
...
+- To measure the health and performance of Temporal-backed applications and key business processes.
+- To measure the health and performance of Temporal infrastructure and user provided infrastructure in the form of Temporal Workers and Temporal Clients.
+
+When measuring the performance of Temporal-backed applications and key business processes, you should rely on Temporal SDK metrics as a source of truth.
+This is because Temporal SDKs provide visibility from the perspective of your application, not from the perspective of the Temporal Service. 
+
+SDK metrics are for monitoring individual workers and your code's behavior.
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/metrics/index.mdx:39]: s/are for monitoring/monitor/

Same for the next line.

Removes passive voice, simplifies wording.

File: docs/production-deployment/cloud/metrics/index.mdx [hunk 6]
```diff
@@ -27,8 +27,18 @@ tags:
...
+Cloud metrics are for monitoring Temporal behavior.
+When used together, Temporal Cloud and SDK metrics help users measure the health and performance of their full Temporal infrastructure, including the Temporal Cloud Service and user supplied Temporal Workers.
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/metrics/index.mdx:41]: consider "When used together, TC and SDK metrics measure the health and performance of the full..."

Missing a hyphen between user and supplied.

File: docs/production-deployment/cloud/metrics/index.mdx [hunk 7]
```diff
@@ -27,8 +27,18 @@ tags:
...
+- To measure the health and performance of Temporal infrastructure and user provided infrastructure in the form of Temporal Workers and Temporal Clients.
+
+When measuring the performance of Temporal-backed applications and key business processes, you should rely on Temporal SDK metrics as a source of truth.
+This is because Temporal SDKs provide visibility from the perspective of your application, not from the perspective of the Temporal Service. 
+
+SDK metrics are for monitoring individual workers and your code's behavior.
+Cloud metrics are for monitoring Temporal behavior.
+When used together, Temporal Cloud and SDK metrics help users measure the health and performance of their full Temporal infrastructure, including the Temporal Cloud Service and user supplied Temporal Workers. 
 
 Metrics for all Namespaces in your account are available from your metrics endpoint.
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/metrics/index.mdx:43]: I cry at the passive voice but this is not part of your patch.

**Review by fairlydurable — APPROVED** [review]
Left comments. Unblocked this moving forward.

Thank you for working on this ticket.

File: docs/production-deployment/cloud/metrics/index.mdx (hunk 3 above)
Comment (axfelix) [reply docs/production-deployment/cloud/metrics/index.mdx:36]: Hm, does it contradict? The point is that SDK metrics are useful for measuring how your own applications perform -- it reads well to me. How can I make it clearer?

File: docs/production-deployment/cloud/metrics/index.mdx (hunk 2 above)
Comment (axfelix) [reply docs/production-deployment/cloud/metrics/index.mdx:30]: True, but I like reinforcing it here.

File: docs/production-deployment/cloud/metrics/index.mdx (hunk 1 above)
Comment (axfelix) [reply docs/production-deployment/cloud/metrics/index.mdx:3]: To be honest, the title casing on these pages seemed weird to me -- I didn't check the capitalize list.

File: docs/production-deployment/cloud/metrics/index.mdx (hunk 3 above)
Comment (axfelix) [reply docs/production-deployment/cloud/metrics/index.mdx:36]: @jlacefie We chatted about this and docs team sounds like they'd like one more clarifying sentence here -- my reading is "If you want to know how your applications behave and how they experience latency -- the SDK metrics are the right ones to use.
 Cloud/Service metrics are more authoritative, but SDK metrics are more faithful to how your applications are actually behaving." Is that OK with you?

**Pull request merged**